src/components/assets/Assets.vue

```diff
@@ -167,6 +156,20 @@ export default defineComponent({
       window.removeEventListener(event, handler);
     });
 
+    const isDarkTheme = computed<boolean>(() => store.getters['general/theme'] === 'DARK');
```

Comment:
We have this line in so many places in our codebase. What do you think moving the line to some hook? It doesn't need to be in this PR

Reply:
Let me work on this in another PR. I'll do it next week